src/agent/src/main.c

```diff
@@ -28,10 +28,7 @@
 #include <azure_c_shared_utility/shared_util_options.h>
 #include <azure_c_shared_utility/threadapi.h> // ThreadAPI_Sleep
 #include <ctype.h>
-#ifndef ADUC_PLATFORM_SIMULATOR // DO is not used in sim mode
```

Comment:
as above.

Reply:
Will remove Agent dependency on DO in other PR